src/xdp/program.c

```diff
@@ -664,7 +664,7 @@ static const ebpf_program_data_t EbpfXdpProgramData = {
     .context_create = XdpCreateContext,
     .context_destroy = XdpDeleteContext,
     .required_irql = DISPATCH_LEVEL,
-    .capabilities = {.supports_context_header = TRUE},
+    .capabilities = {0},
```

Comment:
please just remove this line - implicit zero init is most concise and future-proof for more breaking changes :)